code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -285,7 +286,10 @@
 			if(SSticker.mode && SSticker.mode.is_in_endgame)
 				round_finished = MODE_INFESTATION_X_MINOR //Evacuation successfully took place.
 			else
+				SSticker.roundend_check_paused = TRUE
```

Comment:
Make the setter work w/ forcing instead of just toggling and make this use it too